svir/dialogs/viewer_dock.py

```diff
@@ -831,22 +929,49 @@ def write_export_file(self, filename):
                         line += "," + ",".join([
                             str(value) for value in values])
                     csv_file.write(line + os.linesep)
+            elif self.output_type == 'agg_curves-rlzs':
+                num_rlzs = self.agg_curves['array'].shape[1]
+                rlzs_str = ['rlz-%s' % rlz for rlz in range(num_rlzs)]
+                # write header
+                line = 'return_period,' + ','.join(rlzs_str)
+                csv_file.write(line + os.linesep)
+                for i, return_period in enumerate(
+                        self.agg_curves['return_periods']):
+                    values = self.agg_curves['array'][self.current_loss_type]
+                    line = str(return_period) + "," + ",".join(
+                        [str(value) for value in values[i]])
```

Comment:
Don't you have a generic utility to write CSV files? Even using the csv module in the standard library could be an option.

Reply:
I agree it would be cleaner to use the csv module, although I am not so sure it would actually simplify things. But I guess it would at least improve readability.

Reply:
Done